locales/en.default.schema.json

```diff
@@ -1195,9 +1195,6 @@
           },
           "label": "Desktop content alignment"
         },
-        "color_scheme": {
-          "info": "Visible when container displayed."
-        },
```

Comment:
Ideally we go thru and remove these from all the translated files, though it technically won't hurt too much.